Makefile

```diff
@@ -5,6 +5,10 @@
 
 VERSION:=$(shell jq -r .version package.json)
 UNCRUSTIFY:=uncrustify
+TTY:=-it
+ifdef NO_TTY
+	TTY:=-t
```

Comment:
Doesn't this still enable tty, just not interactively?

```console
$ docker run --help
...
  -t, --tty                            Allocate a pseudo-TTY
...
```

Reply:
It only activates the terminal output and removes the interactivity which was causing the issue.
Interestingly enough, when using `-it` on CircleCI the error does not happen, only on Github Actions.
But it is good to have this option just in case someone needs it.

Reply:
Is there any reason to not remove `-it` entirely? I don't think we need to supply any interactive input for the `docker` invocation in these Makefile targets.

Reply:
We need it for local testing.